docs/source/development-guide/style-guide/git-and-github-workflow.rst

```diff
@@ -59,26 +64,33 @@ well.*
 
    .. code-block:: bash
 
-       pip install .[dev]
-       pre-commit install
+       pip3 install pre-commit
+
+   If this results in errors, check the correct form of pip is being employed using ``which pip``.
 
 #. Retrieve the ``upstream`` ``dev`` branch:
 
    .. code-block:: bash
 
        git fetch upstream
-       git checkout -b dev upstream/dev
+       git checkout -b dev upstream-dev
+
+   To check this step, run ``git status``. The branch name should be ``upstream-dev``.
```

Comment:
Since the `dev` branch is the default branch of the repo, it automatically 'comes down' when you clone the repo, so I think this whole section could be removed actually.

Reply:
Would love to ask more about this on Monday.